core/templates/base-components/base-content.component.rtl.css

```diff
@@ -0,0 +1,82 @@
+/*
+  NOTE to developers: Only the '.css' files should be edited. The styling should
```

Comment:
Ditto re clarifying what a '.css' file is (see above).

Reply:
Done